### PR TITLE
Chore: Migrate more usage of v1 themes to v2

### PR DESCRIPTION
**Pull request by kaydelaney**
More v1 -> v2 theme migrations. Not many left now!

### CODE REVIEW

File: public/app/core/components/Layers/LayerDragDropList.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export const LayerDragDropList = <T extends LayerElement>({
   onNameChange,
   verifyLayerNameUniqueness,
 }: LayerDragDropListProps<T>) => {
-  const style = styles(config.theme);
+  const style = styles(config.theme2);
```
Comment (dprokop) [inline public/app/core/components/Layers/LayerDragDropList.tsx:41]: `useStyles2` instead? It's a function component :)

**Review by dprokop — APPROVED** [review]
I think one applicable useStyles2 slipped through, other than that great.

File: public/app/core/components/Layers/LayerDragDropList.tsx [hunk 2]
```diff
@@ -127,50 +127,50 @@ LayerDragDropList.defaultProps = {
...
   `,
   row: css`
-    padding: ${theme.spacing.xs} ${theme.spacing.sm};
-    border-radius: ${theme.border.radius.sm};
-    background: ${theme.colors.bg2};
-    min-height: ${theme.spacing.formInputHeight}px;
+    padding: ${theme.spacing(0.5, 1)};
+    border-radius: ${theme.shape.borderRadius(1)};
+    background: ${theme.colors.background.secondary};
+    min-height: ${theme.spacing.gridSize * 4}px;
```
Comment (ashharrison90) [inline public/app/core/components/Layers/LayerDragDropList.tsx:138]: could we just do `${theme.spacing(4)}` here - is there any difference? 🤔

**Review by ashharrison90 — APPROVED** [review]
lgtm 🙌 

just one minor nit

**Review by mckn — APPROVED** [review]
LGTM! Agree with the nits mentioned by @dprokop and @ashharrison90

**Pull request merged**